apps-rendering/src/components/liveBlocks.tsx

```diff
@@ -38,19 +38,14 @@ const LiveBlocks: FC<LiveBlocksProps> = ({ blocks, format }) => {
 			{/* Accordion? */}
 			{blocks.map((block) => {
 				const blockLink = `?page=with:block-${block.id}#block-${block.id}`;
-				const blockFirstPublished = pipe(
-					block.firstPublished,
-					map(Number),
-					toNullable,
-				);
 
 				return (
 					<LiveBlockContainer
 						key={block.id}
 						id={block.id}
 						format={format}
 						blockTitle={block.title}
-						blockFirstPublished={blockFirstPublished}
+						blockFirstPublished={Number(block.firstPublished)}
```

Comment:
Why is this turned into a number?

Reply:
Good question. There's some previous discussion on this: https://github.com/guardian/dotcom-rendering/pull/3751#discussion_r767876258

Within `LiveBlockContainer`, this field gets converted to a `Date` here:

https://github.com/guardian/dotcom-rendering/blob/4c1ad4105278ba6c92e5cca5aae98c79b788c103/common-rendering/src/components/FirstPublished.tsx#L31

and used as a plain number here:

https://github.com/guardian/dotcom-rendering/blob/4c1ad4105278ba6c92e5cca5aae98c79b788c103/common-rendering/src/components/FirstPublished.tsx#L63

I think it probably is more accurate to have this field be a `Date`.

Just to be clear though @iainjchambers-guardian, I don't think this is a blocker for this PR, don't worry 🙂

Reply:
Ok, might be better to use the `getTime()` method so it's clearer what's happening at a glace?